static/js/filter.js

```diff
@@ -1,7 +1,95 @@
 (function() {
 	"use strict";
-
     
-	
-	
-})();
\ No newline at end of file
+	const filterEmailAddresses = [];
+	const filterAliasLabels = [];
+	const aliasesWithLabelsCollection = [];
+    const aliasCollection = [];
+    const aliases = document.querySelectorAll(".c-alias");
+
+    const filterLabelTotalCases = document.querySelector(".js-filter-case-total");
+    const filterLabelVisibleCases = document.querySelector(".js-filter-case-visible");
+
+    const filterResetButton = document.querySelector(".js-filter-reset")
+
+    const filterInput = document.querySelector(".js-filter-search-input");
+
+	function filterInputWatcher(e) {
+		const query = e.target.value.toLowerCase();
+
+        // if (query === "") {
+        //     resetFilter();
+        //     return;
+        // }
```

Comment:
be sure to delete commented code before merge.

Reply:
_(This was removed when I added the feature back)_